src/components/properties/controller.ts

```diff
@@ -0,0 +1,72 @@
+import type { Request, Response } from "express";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export const getAllProperties = async (req: Request, res: Response) => {
```

Comment:
have a few comments about this controller. I'll modify the getById method as an example and then refactor the other methods accordingly.